frontend/src/metabase/components/ErrorPages/ErrorDiagnosticModal.tsx

```diff
@@ -90,58 +151,76 @@ export const ErrorDiagnosticModal = ({
         }}
         onSubmit={handleSubmit}
       >
-        <Form>
-          <Text>
-            {t`Select the info you want to include in the diagnostic JSON file.`}
-          </Text>
-          <Stack spacing="md" my="lg">
-            {canIncludeQueryData && (
-              <FormCheckbox name="queryResults" label={t`Query results`} />
-            )}
-            {!!errorInfo.localizedEntityName && (
-              <FormCheckbox
-                name="entityInfo"
-                label={t`${capitalize(
-                  errorInfo.localizedEntityName,
-                )} definition`}
-              />
-            )}
-            <FormCheckbox
-              name="frontendErrors"
-              label={t`Browser error messages`}
-            />
-            {!!errorInfo?.logs && (
-              <>
-                <FormCheckbox
-                  name="backendErrors"
-                  label={t`All server error messages`}
-                />
-                <FormCheckbox name="logs" label={t`All server logs`} />
+        {formik => (
+          <Form>
+            <Text>
+              {t`Select the info you want to include in the diagnostic JSON file.`}
+            </Text>
+            <Stack spacing="md" my="lg">
+              {canIncludeQueryData && (
+                <FormCheckbox name="queryResults" label={t`Query results`} />
+              )}
+              {!!errorInfo.localizedEntityName && (
                 <FormCheckbox
-                  name="userLogs"
-                  label={t`Server logs from the current user only`}
+                  name="entityInfo"
+                  label={t`${capitalize(
+                    errorInfo.localizedEntityName,
+                  )} definition`}
                 />
-              </>
-            )}
-            <FormCheckbox
-              name="bugReportDetails"
-              // eslint-disable-next-line no-literal-metabase-strings -- we're mucking around in the software here
-              label={t`Metabase instance version information`}
-            />
-          </Stack>
-          <Alert variant="warning">
-            {t`Review the downloaded file before sharing it, as the diagnostic info may contain sensitive data.`}
-          </Alert>
-          <Flex gap="sm" justify="flex-end" mt="lg">
-            <Button onClick={onClose}>{t`Cancel`}</Button>
-            <FormSubmitButton
-              variant="filled"
-              leftIcon={<Icon name="download" />}
-              label={t`Download`}
-              color="brand"
-            />
-          </Flex>
-        </Form>
+              )}
+              <FormCheckbox
```

Comment:
Since this is a 1-shot bug report, I think from a ux standpoint, we need to add a `textarea` to describe the bug. I actually found a bug while testing this and reported it.. but nobody would ever have a clue what the bug was from the report


![Screen Shot 2024-11-14 at 7 53 55 AM](https://github.com/user-attachments/assets/10eb355c-5fda-4213-887b-f32c93097329)

Reply:
Yep that's planned!